samples/tab-mail/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-mail-ts",
+    "source": "officeDev",
+    "title": "Tab Mail In Outlook",
```

Comment:
Compose Mail in Outlook using Personal Tab app.

Reply:
fixed